src/platform/Darwin/DnssdContexts.cpp

```diff
@@ -566,16 +566,17 @@ void ResolveContext::DispatchSuccess()
     }
 }
 
-bool ResolveContext::TryReportingResultsForInterfaceIndex(uint32_t interfaceIndex)
+bool ResolveContext::TryReportingResultsForInterfaceIndex(uint32_t interfaceIndex, const std::string & hostname, bool isSRPResolve)
```

Comment:
Maybe make that third arg be isSRPResult to match the naming elsewhere?  I guess I suggested this naming before I realized where these booleans were coming from.

But note: in OnNewInterface below, isFromSRPResolve is the right name.  So don't change it there.  ;)

Reply:
done.